doctest/build.gradle

```diff
@@ -115,11 +102,7 @@ task stopPrometheus() {
         }
     }
 }
-if(getOSFamilyType() != "windows") {
-    stopPrometheus.mustRunAfter startPrometheus
-    startOpenSearch.dependsOn startPrometheus
-    stopOpenSearch.finalizedBy stopPrometheus
-}
```

Comment:
Isn't this still necessary?

startOpenSearch needs startPrometheus because there are some integration tests that use both.
stopPrometheus needs to be run after stopOpenSearch.